File: DataServer/src/main/java/model/Action.java
package model;

public class Action {

    private String creator;
    private String activity;
    private int academic;
    private int financial;
    private int health;
    private int social;
    private int timeCost;

    public Action(String creator, String activity, int academic, int financial, int health, int social, int timeCost) {
        this.creator = creator;
        this.activity = activity;
        this.academic = academic;
        this.financial = financial;
        this.health = health;
        this.social = social;
        this.timeCost = timeCost;
    }

    public String getCreator() {
        return creator;
    }

    public String getActivity() {
        return activity;
    }

    public int getAcademic() {
        return academic;
    }

    public int getFinancial() {
        return financial;
    }

    public int getHealth() {
        return health;
    }

    public int getSocial() {
        return social;
    }

    public int getTimeCost() {
        return timeCost;
    }
}

File: WebAPI/IncWebAPI/Model/Action.cs
using System.Text.Json.Serialization;

namespace WebAPI.Model
{
    public class Action
    {
        public string Creator { get; }
        public string Activity { get; }
        public int Academic { get; }
        public int Financial { get; }
        public int Health { get; }
        public int Social { get; }
        public int TimeCost { get; }

        [JsonConstructor]
        public Action(string creator, string activity, int academic, int financial, int health, int social,
            int timeCost)
        {
            Creator = creator;
            Activity = activity;
            Academic = academic;
            Financial = financial;
            Health = health;
            Social = social;
            TimeCost = timeCost;
        }
    }
}